schemas/gazette-notice-schema.json

```diff
@@ -0,0 +1,171 @@
+      "$ref": "includes/entity.json"
+    },
+    "other_attributes": {
+      "description": "Any other structured data from the notice ",
+      "type": "object"
+    }
+  },
+  "additionalProperties": false,
+  "required": [
+    "issue",
+    "identifier"
+  ],
+  "definitions": {
+    "publisher": {
+      "title": "Publisher",
+      "description": "A gazette publisher",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The name of the publisher",
+          "type": "string"
+        },
+        "url": {
+          "description": "The URL of the publisher",
+          "type": "string",
+          "format": "uri"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "name"
+      ]
+    },
+    "publication": {
+      "title": "Publication",
+      "description": "A gazette",
+      "type": "object",
+      "properties": {
+        "publisher": {
+          "description": "The publisher of the gazette",
+          "$ref": "#/definitions/publisher"
+        },
+        "jurisdiction_code": {
+          "description": "The jurisdiction covered by the gazette",
+          "type": "string"
+        },
+        "title": {
+          "description": "The title of the gazette",
+          "type": "string"
+        },
+        "url": {
+          "description": "The URL of the gazette",
+          "type": "string",
+          "format": "uri"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "publisher"
+      ]
+    },
+    "issue": {
+      "title": "Issue",
+      "description": "An issue of a gazette",
+      "type": "object",
+      "properties": {
+        "publication": {
+          "description": "The gazette of which the issue is a part",
+          "$ref": "#/definitions/publication"
+        },
+        "identifier": {
+          "description": "The identifier of the issue",
+          "type": [
+            "string",
+            "integer"
+          ]
+        },
+        "title": {
+          "description": "The title of the issue",
+          "type": "string"
+        },
+        "edition_id": {
+          "description": "The identifier of the edition of which the issue is a part",
+          "type": [
+            "string",
+            "integer"
+          ]
+        },
+        "page_range": {
+          "description": "The page range of the issue",
+          "type": "string",
+          "pattern": "^[0-9]*(-[0-9]*)?$"
+        },
+        "date_published": {
+          "description": "The date of publication of the issue",
+          "type": "string",
+          "pattern": "^[0-9]{4}(-[0-9]{2}){2}$"
+        },
+        "url": {
+          "description": "The URL of the issue",
+          "type": "string",
+          "format": "uri"
+        },
+        "media_type": {
+          "description": "The media type of the URL resource",
+          "type": "string"
+        },
+        "retrieved_at": {
```

Comment:
This should be a child of the root, as it's when the gazette notice was published (it may take several minutes to iterate through an issue). 
Also we should use the same format patterns as elsewhere for datetimes (e.g. https://github.com/openc/openc-schema/blob/master/schemas/company-schema.json#L32)
@sebbacon will confirm, but I think we also need source_url and sample_date in the schema to allow this data to be ingested by the data pipeline